src/Service/CacheService.php

```diff
@@ -228,7 +284,12 @@ public function warmup(array $config = []): int
                     continue;
                 }
             }
-        }
+
+            if ($bundleToCache !== null) {
+                isset($this->style) === true && $this->style->writeln('When caching a specific bundle, will only cache the objects for now. ');
```

Comment:
I would expect the removeDataFromCache to run as well, but only for objects from this bundle.
For now oke, but it feels weird to not do remove as well,